src/probnum/diffeq/odesolver.py

```diff
@@ -28,14 +28,20 @@ def solve(self, firststep, steprule, **kwargs):
         """
         t, current_rv = self.initialise()
         times, rvs = [t], [current_rv]
-        stepsize = firststep
+        stepsize = steprule.firststep
```

Comment:
All steprules have a firststep now, but for e.g. constant steps, this is chosen automatically.

Reply:
I'm still unsure if `firststep` should be part of steprule, or if it should just be initialized via `step` (in `probsolve_ivp`) but still explicitly passed. I have the impression though that it might be more productive to talk about this directly instead of writing?

Reply:
To be honest, I dont understand what you are talking about...

Reply:
Sorry if my statement was confusing!

My proposal would be to revert to explicitly passing `firststep`, which, when the solver is called via `probsolve_ivp`, is set to `step`. But this relates also to how the steprule is chosen (inferred from `step` and `tol` vs an explicit argument). Does that make more sense now?

Reply:
Yes, thanks for clarifying. I think I disagree with this, sorry to say this so openly... One of the reasons is that for random-step solvers (do you remember the abdulle/garegnani paper?) it is easiest to randomise steprules instead of solvers. The handling of firststep is then best kept in the steprule. But also in general, I think the first step that the solver takes fits within the scope of steprule (much more than atol/rtol do, IMO). What happens in probsolve_ivp is a different matter to me, I think it may be fun to reconceptualise the code in there soon.